.github/workflows/tests.yml

```diff
@@ -48,26 +48,17 @@ jobs:
       matrix:
         include:
           - name-suffix: "(Minimum Versions)"
-            os: ubuntu-20.04
-            python-version: '3.10'
+            os: ubuntu-22.04
+            python-version: '3.11'
```

Comment:
Is it decided that we will drop Python 3.10 for next release? April 5 2025 based on NEP29, but experience says that it is a bit relaxed.